cdk_lib/dpdash-cdk-stack.ts

```diff
@@ -1,17 +1,16 @@
-import * as cdk from 'aws-cdk-lib';
-import * as ec2 from "aws-cdk-lib/aws-ec2";
-import * as ecs from "aws-cdk-lib/aws-ecs";
-import * as ssm from "aws-cdk-lib/aws-ssm";
-import * as ses from "aws-cdk-lib/aws-ses";
-import * as ecr from "aws-cdk-lib/aws-ecr";
-import * as iam from "aws-cdk-lib/aws-iam";
-import * as docdb from "aws-cdk-lib/aws-docdb";
-import * as route53 from "aws-cdk-lib/aws-route53";
-import * as certificate_manager from "aws-cdk-lib/aws-certificatemanager";
-import * as ecs_patterns from "aws-cdk-lib/aws-ecs-patterns";
-import * as secrets_manager from "aws-cdk-lib/aws-secretsmanager";
-import { Construct } from 'constructs';
-
+import * as cdk from 'aws-cdk-lib'
```

Comment:
These changes are going to conflict when you rebase, I would recommend redoing them on the latest version of this file rather than trying to hand-fix the diff.